File: src/main/java/kr/nyamnyam/model/domain/ReceiptModel.java
package kr.nyamnyam.model.domain;

import lombok.Data;
import org.springframework.stereotype.Component;

import java.util.Date;


@Data
@Component
public class ReceiptModel {

    private Long id;
    private Long price;
    private String name;
    private String menu;
    private String userId;
    private String date;
    private Date entryDate;


}

File: src/app/model/receipt.model.ts
export interface ReceiptModel {
    id: number,
    price: number,
    name: string,
    menu: string,
    userId: number,
    date: string

}

export const initialReceipt: ReceiptModel = {
    id: 0,
    price: 0,
    name: '',
    menu: '',
    userId: 0,
    date: ''
}